inclusive-design/wecount.inclusivedesign.ca
# feat: implement design refresh

## Description

Implements new design updates including full-width video on the home page, a "Subscribe" button in the footer and new navigation styling.

## Steps to test

1. Verify the styling updates are present and look acceptable
2. Test the home page embedded YouTube video at a variety of screen widths to ensure it stays full-width at all times*
3. Verify the Subscribe button is present at the top of the footer and working properly

_*below viewport widths of approximately 250 pixels, things start to get weird for all site content, not just the video._

## Additional information

This PR also solves an issue which caused the build script to fail in PowerShell.

## Related issues

Some issues are introduced or represent incomplete work from this PR:
- #649
- #651
- #652
